src/converter/style.py

```diff
@@ -404,7 +416,10 @@ def context_settings(fig_node: dict) -> ContextSettings:
     blend_mode = BLEND_MODE[fig_node.get("blendMode", "NORMAL")]
     opacity = fig_node.get("opacity", 1)
 
-    if blend_mode == BlendMode.NORMAL and opacity == 1:
+    # Figma's default blend mode is pass-through, but its not expressed as a
+    # value in the fig model. When "NORMAL" is set explicity we need to tweak Sketch'
+    # opacity to avoid pass-through.
+    if fig_node.get("blendMode") == "NORMAL" and opacity == 1:
         # Sketch interprets normal at 100% opacity as pass-through
         opacity = 0.99
```

Comment:
So my testing leads me to believe that when a figma entity's blend mode is set to `pass-through` the blend mode is omitted from the `fig` model as its the default value. 

This means we can test for Pass-through vs Normal more reliably by checking the fig node, and if blend mode is set to "NORMAL" explicitly, then we can match the behaviour in Sketch by dropping to 99% opacity.